adapters/repos/db/vector/hnsw/delete.go

```
@@ -108,6 +109,15 @@ func (h *hnsw) DeleteMulti(docIDs ...uint64) error {
 			if err := h.Delete(id); err != nil {
 				return err
 			}
+			idBytes := make([]byte, 8)
+			binary.BigEndian.PutUint64(idBytes, id)
+			if err := h.store.Bucket(h.id + "_mv_mappings").Delete(idBytes); err != nil {
+				h.logger.WithFields(logrus.Fields{
```

Asking just in case: is it safe to ignore this error?